### PR TITLE
Replacing SSL/TLS certificate and key

### PR DESCRIPTION
**Pull request by tphee**

### CODE REVIEW

File: stories/topics/proc-gcp-additional-configs-update-admin-password.adoc [hunk 1]
```diff
@@ -14,5 +14,5 @@ Follow these steps to change the administrator password for both {ControllerName
 . Replace the running {PlatformNameShort} VM instances to use the new administrator password.
 .. Navigate to the *GCP VM Instances* console.
 .. Identify and delete one {ControllerName} VM instance and one {HubName} VM instance for the {PlatformNameShort} deployment.
```
Comment (chrisahl) [inline stories/topics/proc-gcp-additional-configs-update-admin-password.adoc:16]: It seems that you are only removing one of the two controllers.  Wouldn't this cause one controller with old pwd and one with new?

File: stories/topics/ref-aws-additional-configs-update-admin-password.adoc [hunk 2]
```diff
@@ -13,5 +13,5 @@ Follow these steps to change the Administrator password for both {ControllerName
 . Replace the running {PlatformNameShort} EC2 instances to use the new Administrator password.
 .. Navigate to the AWS EC2 Instances console.
 .. Identify and Terminate one {ControllerName} instance and one {HubName} instance for the CloudFormation Stack.
```
Comment (chrisahl) [inline stories/topics/ref-aws-additional-configs-update-admin-password.adoc:15]: It seems that you are only removing one of the two controllers. Wouldn't this cause one controller with old pwd and one with new?

File: stories/topics/proc-gcp-additional-configs-update-admin-password.adoc (hunk 1 above)
Comment (tphee) [reply stories/topics/proc-gcp-additional-configs-update-admin-password.adoc:16]: Only have to update the password in the database, so one controller is enough to perform the update.

File: stories/topics/ref-aws-additional-configs-update-admin-password.adoc (hunk 2 above)
Comment (tphee) [reply stories/topics/ref-aws-additional-configs-update-admin-password.adoc:15]: Only have to update the password in the database, so one controller is enough to perform the update.

**Review by chrisahl — APPROVED** [review]

**Comment by zkayyali812** [comment]
Hey @tphee,
Doc updates look great!

Small nit - 
image
On the AWS doc, can we move the highlighted sections to be under the `Additional Configurations` chapter? This way these sections are under the same chapters for both AWS/GCP.

**Comment by tphee** [comment]
@zkayyali812 I've moved highlighted sections to under the Additional Configurations chapter

**Comment by zkayyali812** [comment]
@tphee One more nit, im sorry - 

image
Can we put the section `6.4.1. Securing the controller load balancer` directly under the additional configurations chapter? So it is would be number as 6.5? This way it would be visible directly under Additional Configurations, as opposed to being nested under chapter 6.4.

**Comment by tphee** [comment]
image

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
+[id="proc-gcp-additional-configs-replace-tls-certificate"]
+
+= Replacing {ControllerName} and {HubName} VM instances SSL/TLS certificate and key
+
+By default, VM instances are secured with a self-signed SSL/TLS certificate with a validity period of ten years.
+Once the certificate expires or you want VM instances to use your own certificate, you are required to replace the SSL/TLS certificate and key.
```
Comment (ianf77) [inline stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc:6]: When the...

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+.Procedure
+. Locate the correct secret for the {PlatformNameShort} deployment by opening the GCP Secrets Manager console and selecting the secret with the name _<deployment_name>-pulp_cert_.
+. Select *NEW VERSION* to add a new version.
+. Enter new SSL/TLS certificate value.
+. Check *Disable all past versions*.
```
Comment (ianf77) [inline stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc:12]: Check the *Disable all past versions* checkbox.

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+. Click btn:[ADD NEW VERSION].
+. Select the secret with the name _<deployment_name>-pulp_key_.
+. Select *NEW VERSION* to add a new version.
+. Enter new SSL/TLS key value.
+. Check *Disable all past versions*.
```
Comment (ianf77) [inline stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc:17]: As before, do you need to do this twice? Are these checkboxes on the same page?

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+. Click btn:[ADD NEW VERSION].
+. Replace the running {PlatformNameShort} VM instances to use the new SSL/TLS certificate and key.
+.. Navigate to the *GCP VM Instances* console.
+.. Identify and delete all {ControllerName} and {HubName} VM instances for the {PlatformNameShort} deployment.
+.. Wait for the {ControllerName} and {HubName} Instance groups to create new VM instances.
+. The new certificate is in used when the new {ControllerName} and {HubName} VM instances reach a _Running_ Instance State.
```
Comment (ianf77) [inline stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc:23]: is in use when

File: stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc [hunk 7]
```diff
@@ -0,0 +1,21 @@
+[id="ref-aws-additional-configs-replace-tls-certificate"]
+
+= Replacing {ControllerName} and {HubName} EC2 instances SSL/TLS certificate and key
+
+By default, EC2 instances are secured with a self-signed SSL/TLS certificate with a validity period of ten years.
+Once the certificate expires or you want EC2 instances to use your own certificate, you are required to replace the SSL/TLS certificate and key.
```
Comment (ianf77) [inline stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc:6]: When

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc [hunk 8]
```diff
@@ -0,0 +1,23 @@
+[id="proc-gcp-additional-configs-replace-tls-certificate"]
+
+= Replacing {ControllerName} and {HubName} VM instances SSL/TLS certificate and key
+
+By default, VM instances are secured with a self-signed SSL/TLS certificate with a validity period of ten years.
+Once the certificate expires or you want VM instances to use your own certificate, you are required to replace the SSL/TLS certificate and key.
+
+.Procedure
+. Locate the correct secret for the {PlatformNameShort} deployment by opening the GCP Secrets Manager console and selecting the secret with the name _<deployment_name>-pulp_cert_.
```
Comment (ianf77) [inline stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc:9]: . Navigate to the GCP Secrets Manger Console.
. Locate the secret for the {PlatformNameShort deployment, with the name <deployment_name>-pulp-cert. (wrap the whole thing in back ticks and <deployment_name> in underscores.)

File: stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+.Procedure
+. Locate the correct secret for the CloudFormation Stack by opening the AWS Secrets Manager console and selecting the secret with the name _<stack_name>-pulp_cert_.
```
Comment (ianf77) [inline stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc:9]: Navigate to the AWS Secrets Manager console.
Locate the secret for the CloudFormation Stack with the name <stack_name>-pulp_cert (markup as before)

File: stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+. Select *Retrieve secret value* to view the secret value.
+. Select *Edit* and provide new SSL/TLS certificate value.
+. Click btn:[Save].
+. Select the secret with the name _<stack_name>-pulp_key_.
+. Select *Retrieve secret value* to view the secret value.
+. Select *Edit* and provide new SSL/TLS key value.
+. Click btn:[Save].
+. Replace the running {PlatformNameShort} EC2 instances to use the new SSL/TLS certificate and key.
```
Comment (ianf77) [inline stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc:17]: Change the running...

File: stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+.. Navigate to the AWS EC2 Instances console.
+.. Identify and Terminate all {ControllerName} and {HubName} instances for the CloudFormation Stack.
+.. Wait for the {ControllerName} and {HubName} Auto Scaling Groups to create new EC2 instances.
+. The new certificate is in used when the new {ControllerName} and {HubName} EC2 instances reach a _Running_ instance state.
```
Comment (ianf77) [inline stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc:21]: is in use...

**Review by ianf77 — COMMENTED** [review]
A few comments

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc (hunk 3 above)
Comment (tphee) [reply stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc]: Fixed

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc (hunk 5 above)
Comment (tphee) [reply stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc]: Two different secrets.  

image

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc (hunk 6 above)
Comment (tphee) [reply stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc]: Fixed

File: stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc (hunk 7 above)
Comment (tphee) [reply stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc]: Fixed

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc (hunk 8 above)
Comment (tphee) [reply stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc]: Fixed

File: stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc (hunk 9 above)
Comment (tphee) [reply stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc]: image

File: stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+. Select *Retrieve secret value* to view the secret value.
+. Select *Edit* and provide new SSL/TLS certificate value.
+. Click btn:[Save].
+. Select the secret with the name _<stack_name>-pulp_key_.
+. Select *Retrieve secret value* to view the secret value.
+. Select *Edit* and provide new SSL/TLS key value.
+. Click btn:[Save].
+. Replace the running {PlatformNameShort} EC2 instances to use the new SSL/TLS certificate and key.
+.. Navigate to the AWS EC2 Instances console.
+.. Identify and Terminate all {ControllerName} and {HubName} instances for the CloudFormation Stack.
```
[the remark this answers is not in the document]
Comment (tphee) [reply stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc]: Fixed

File: stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc (hunk 11 above)
Comment (tphee) [reply stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc]: Fixed

File: stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc (hunk 10 above)
Comment (tphee) [reply stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc]: Fixed

File: stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+. Select *NEW VERSION* to add a new version.
+. Enter new SSL/TLS certificate value.
+. Check *Disable all past versions*.
+. Click btn:[ADD NEW VERSION].
+. Select the secret with the name _<deployment_name>-pulp_key_.
+. Select *NEW VERSION* to add a new version.
+. Enter new SSL/TLS key value.
+. Check *Disable all past versions*.
+. Click btn:[ADD NEW VERSION].
+. Replace the running {PlatformNameShort} VM instances to use the new SSL/TLS certificate and key.
```
[the remark this answers is not in the document]
Comment (tphee) [reply stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc]: Fixed

**Pull request merged**